math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -26,28 +26,25 @@
 #include <ROOT/TypeTraits.hxx>
 
 #include <algorithm>
-#include <cmath>
 #include <numeric> // for inner_product
 #include <sstream>
 #include <stdexcept>
 #include <type_traits>
 #include <vector>
 #include <utility>
 
-#ifdef R__HAS_VDT
-#include <vdt/vdtMath.h>
-#endif
-
-// TODO: Figure out best way to get pi on windows
-// TODO: ROOT::Internal::VecOps::pi is set to float because of narrowing errors on windows.
-#if defined(_MSC_VER)
+#ifdef _WIN32
+// cmath does not expose M_PI on windows
 #include <math.h>
+#else
```

Comment:
Keeping both `#include <cmath>` and `#include <math.h>` should not hurt (I think)